src/Kiota.Builder/Configuration/GenerationConfiguration.cs

```diff
@@ -142,6 +142,11 @@ public bool? IncludePluginExtensions
         get; set;
     }
 
+    public bool NoWorkspace
+    {
+        get; set;
+    }
+
```

Comment:
This one also needs to be updated
https://github.com/microsoft/kiota/blob/be6e7d048e935f07a74a932688f99c178fc72345/src/kiota/KiotaConfigurationExtensions.cs#L67

Reply:
fixed